tests/playwright-tests/tests/themes.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test } from '@playwright/test';
+import environment from '../utils/env';
+import AzureLoginPage from '../admin/azpage/AzureLoginPage';
+import AdminPage from '../admin/pages/AdminPage';
+import ThemesPage from '../admin/pages/ThemesPage';
+import CreateThemePage from '../admin/pages/CreateThemePage';
+import generateRunIdentifier from '../utils/generateRunIdentifier';
+
+test.describe('Verify the end to end functionality of themes and topics', () => {
+  let adminPage: AdminPage;
+  let themesPage: ThemesPage;
+  let createThemePage: CreateThemePage;
+  let azPage: AzureLoginPage;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto(environment.ADMIN_URL);
+    adminPage = new AdminPage(page);
+    themesPage = new ThemesPage(page);
+    createThemePage = new CreateThemePage(page);
+    azPage = new AzureLoginPage(page);
+
+    await adminPage.clickSignIn();
+    await azPage.doSignIn();
+  });
+
+  test('Verify that themes are being created and displayed in the themes home screen', async () => {
+    await adminPage.manageThemesTopicLink.click();
+    await themesPage.createThemeLink.click();
+
+    const uiThemeName = generateRunIdentifier();
+    const uiThemeTitle = `Theme${uiThemeName}`;
```

Comment:
Wouldn't this generate something like the following? `ThemeUI Test2024-01-17 12:00:00`

This doesn't look right for a few reasons:

- It looks like inconsistent from current conventions for naming high level things like themes in the UI tests
- There isn't sufficient spacing between the parts due to how the interpolations are arranged in the string

Would suggest we do something like the following instead: `UI Test - Test theme - 2024-01-17 12:00:00`

This is neater and follows our naming convention better.

Check out my later comment in `generateRunTimeIdentifier.ts` on how we could abstract this.